index.html

```diff
@@ -2983,7 +2973,7 @@ <h2 class="code-line" data-line-start=9 data-line-end=10><a id="Companies_that_r
 					<td>14.07.22</td>
 				</tr>
 				<tr>
-					<td style="text-align:center">TikTok</td>
+					<td style="text-align:center"><a href="https://jobs.bytedance.com/en/position?keywords=&category=6704215862603155720&location=CT_163&project=&type=1&job_hot_flag=&current=1&limit=10">TikTok</a></td>
```

Comment:
I think it would be great if you can rename this to Bytedance (TikTok), so both names would be there!